File: TaskManagerZSU/ProjectClasses/Project.cs
using DataLayer.UserClasses;

using TaskManagerZSU;

namespace DataLayer.ProjectClasses
{
    public class Project : Entry
    {
        public string Label { get; set; }
        public List<Report> Reports { get; set; }
        public List<Unit> UnitsCreators { get; set; } 
        public List<User> Admins { get; set; }
        public User DefaultExecutor { get; set; }
        public User HeadOfProject { get; set; }
    }
}

File: WebView/src/app/Classes/Tasks/project.ts
import { Task } from "./task";
import { Unit } from "../Unit";
import { User } from "../user";

export class Project{
    constructor(public Id : number, 
        public Label : string, 
        public Admins : User[], 
        public UnitsCreators : Unit[], 
        public DefaultExecutor : User, 
        public HeadOfProject : User, 
        public Reports : Task[]){}
}